docs/rules/max-statements-per-line.md

```diff
@@ -0,0 +1,67 @@
+# Limit Maximum Number of Statements Per Line (max-statements-per-line)
+
+The `max-statements-per-line` rule allows you to specify the maximum number of statements allowed per line.
+
+```js
+function foo() {
+  var bar = 1; // one statement
+  var baz = 2; var corge = 3; // two statements
+  var qux = 4; var quux = 5; var waldo = 6; // three statements
+}
+```
+
+## Rule Details
+
+This rule allows you to configure the maximum number of statements allowed per line.  The default is 1.
+
+## Options
```

Comment:
Nope, `options` still stays as level 2, however, each individual option should be level 3 heading now.